bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx90-en.html

```diff
@@ -0,0 +1,129 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}What’s new with Firefox{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_90') }}
+{% endblock %}
+
+{% block experiments %}
+  {% if switch('firefox-whatsnew90-experiment', ['en-US', 'en-CA']) %}
+    {{ js_bundle('firefox_whatsnew90_experiment') }}
+  {% endif %}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew90' %}
+
+{% set utm_campaign = 'wnp90' %}
+
+{% if variant == '1' %}
+  {% set utm_content= 'v1' %}
+{% elif variant == '2' %}
+  {% set utm_content = 'v2' %}
+{% else %}
+  {% set utm_content = 'all' %}
+{% endif %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+      <img class="wnp-main-image" src="{{ static('protocol/img/logos/pocket/logo-word-hor.svg') }}" width="220" alt="Pocket">
+      <h1 class="wnp-main-title">Your save button for the internet</h1>
+      <p class="wnp-main-tagline">Click the <img class="inline-icon" src="{{ static('img/firefox/whatsnew/whatsnew90/pocket-outline.svg') }}" width="16" height="16" alt="Save to Pocket"> button in your Firefox toolbar anytime you see something you want to save for later, like an article, video — even a tweet. It’ll go straight to your library in Pocket.</p>
+
+      <div class="wnp-main-cta">
+      {% if variant == '2' %}
+        <a class="mzp-c-button mzp-t-product" href="https://getpocket.com/signup/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ utm_campaign }}&utm_content={{ utm_content }}" data-cta-text="Sign up for Pocket" data-cta-position="primary" data-cta-type="button">Sign up for Pocket</a>
+      {% else %}
+        {{ pocket_fxa_button(entrypoint=_entrypoint, button_text='Activate Pocket in Firefox', include_metrics=False, optional_parameters={'s': _entrypoint, 'utm_campaign': utm_campaign, 'utm_content': utm_content + 'fxa'}, optional_attributes={'data-cta-text': 'Activate Pocket in Firefox', 'data-cta-type': 'button', 'data-cta-position': 'primary'}) }}
```

Comment:
Oh huh, I assumed we wanted to include FxA metrics on these sign-in buttons. If that's not the case (and why you added `include_metrics=False`, then adding `mozilla-fxa-product-button.js` probably isn't required.

Up to you what you want to do here, but otherwise this looks good to me. r+

Reply:
doh, that was a leftover from trying to remove utm_medium. Fixing.